adapter/src/main/kotlin/org/javacs/ktda/adapter/KotlinDebugAdapter.kt

```diff
@@ -270,6 +271,7 @@ class KotlinDebugAdapter(
 			exceptionsPool.clear()
 			converter.variablesPool.clear()
 			converter.stackFramePool.removeAllOwnedBy(args.threadId)
+			(debuggee as JDIDebuggee).resumeVm()
```

Comment:
I think it would be better style to move the `resumeVm` method into the `Debuggee` interface, so that we don't have to downcast (and couple the debug adapter to JDI classes, which are considered to be implementation details at this level).

Reply:
Ok, thank you. I will make this change it in next commit, in which I am trying to fix a deadlock bug from debugging the Spring Boot example.